README.md

```diff
@@ -66,13 +67,10 @@ For most of the tools, the configuration is in the `package.json` to reduce the
 
 If you customize the configuration a lot, you can consider moving them to individual files.
 
-## Deploying using `gh-pages`
+## Manual Deploy using `gh-pages`
```

Comment:
Question: 

I assume that even PRs with names starting in "Draft" or "WIP" will have demo apps created.
Is there any reason to _**not**_ create a demo app?
And if there is, can we have a label that: 

1. Stops the creation of a demo app and
2. Shows that there's no demo: a label like `no-demo-app`, and 
3. A demo app is created when that label is removed, and last
4. Any existing demo app is destroyed when label added.

I can't think of any reason not to create a demo app.

Reply:
thanks for your inputs @mc2.

- in case that your next push and the code/tests failed, the demo app will remain

as for the others, will check the feasibility on those. as for now, these are the base foundation of what we need so far.